registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java

```diff
@@ -301,16 +302,15 @@ private void createIdentifyRequest(List<AbisQueueDetails> abisQueueDetails, Stri
 		AbisIdentifyRequestDto abisIdentifyRequestDto = new AbisIdentifyRequestDto();
 		Flag flag = new Flag();
 		abisIdentifyRequestDto.setId(AbisHandlerStageConstant.MOSIP_ABIS_IDENTIFY);
-		abisIdentifyRequestDto.setVer(AbisHandlerStageConstant.VERSION);
+		abisIdentifyRequestDto.setVersion(AbisHandlerStageConstant.VERSION);
 		abisIdentifyRequestDto.setRequestId(id);
 		abisIdentifyRequestDto.setReferenceId(bioRefId);
-		abisIdentifyRequestDto.setReferenceUrl(url + "/" + bioRefId);
+		abisIdentifyRequestDto.setReferenceUrl("");

```

Comment:
I dont think this is correct. You cant just make setReferenceUrl as "", its conditional, 
1. for insert the reference url will be sent
2. for identify the url will be "".

check spec again and do the changes https://docs.mosip.io/platform/apis/abis-apis#insert